public/blog/2019-06-01-to-2019-12-31.md

```diff
@@ -4,4 +4,4 @@ Popcorn Sales Begin in August! Please take a moment to follow the steps below an
 
 New technology will make this year's popcorn fundraising easier and more fun.
 
-See our [Popcorn Page](http://cub306.local/popcorn/popcorn2019.md)
+See our [Popcorn Page](http://cub306.com/popcorn/popcorn2019.md)
```

Comment:
I'll do a "grep" of the source code and see if any more of these exist. I can do a bulk change if so. Good catch. FYI, when I run the website on my laptop, that is the URL I use.

Reply:
actually we are a .org, I'll fix this.